src/documentation/0012-node-cli-args/index.md

```diff
@@ -91,3 +91,20 @@ This time you need to use double dashes before each argument name:
 ```bash
 node app.js --name=joe
 ```
+
+A complete code example that shows how to create an application using Node CLI Args:
+
+```
+const argvs = process.argv
```

Comment:
Most of the JS code examples use code-style with semicolons, please add them

Reply:
okay, sure